packages/core/src/services/documents/destroyOrSoftDeleteDocuments.ts

```diff
@@ -140,6 +138,18 @@ export async function destroyOrSoftDeleteDocuments(
   },
   transaction = new Transaction(),
 ): Promise<TypedResult<boolean, Error>> {
+  const deleteAllTriggerDocumentsResult =
+    await deleteAllDocumentTriggersFromCommit(
+      {
+        commit,
+        workspace,
+      },
+      transaction,
+    )
```

Comment:
Coordinate with @csansoon this should be fine with what he's doing but check with him.